main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -173,7 +177,27 @@ DataTableView.prototype._renderPagingControls = function(pageSizeControls, pagin
     previousPage.addClass("inaction");
   }
 
-  $('<span>').addClass("viewpanel-pagingcount").html(" " + from + " - " + to + " ").appendTo(pagingControls);
+  var pageControlsSpan = $('<span>').attr("id", "viewpanel-paging-current");
+  
+  var pageInputSize = 20 + (8 * ui.dataTableView._lastPageNumber.toString().length);
+  var currentPageInput = $('<input type="number">')
+    .change(function(evt) { self._onChangeGotoPage(this, evt); })
+    .keydown(function(evt) { self._onKeyDownGotoPage(this, evt); })
+    .attr("id", "viewpanel-paging-current-input")
+    .attr("min", 1)
+    .attr("max", self._lastPageNumber)
+    .attr("required", "required")
+    .val(self._currentPageNumber)
+    .css("width", pageInputSize +"px")
+    .appendTo(pageControlsSpan);
+
+  pageControlsSpan.append(' of ');
```

Comment:
This `' of '` should be localized, and we need to rework the code a bit so that the page numbers that are prepended / appended to the message are inserted in a template instead (so that translators can choose to reorder them if necessary). If this is unclear let me know, I can give examples.

Reply:
Yes. Somewhere, I asked you an example of code using templates. There is the concept of « placeholder » -> « [FIELD] of [SPAN] pages », and the concept of singular/plural: « [FIELD] of [1] page ».

Reply:
See https://github.com/OpenRefine/OpenRefine/issues/1858#issuecomment-636313972

Reply:
> See [#1858 (comment)](https://github.com/OpenRefine/OpenRefine/issues/1858#issuecomment-636313972)

Thanks @wetneb for the pointer. It is now coded and working, FR, EN & ES.

I think I have answered all questions and fulfilled all requests for the PR. Is there anything next?

Regards,
   Antoine

Reply:
@wetneb: K, I fixed the problem by using « page(s) » in french. The problem is only in French (AFAIK) and I described the problem in #1858.

For this issue, everything should be K.

Regards,
   Antoine